data/formats-data.js

```diff
@@ -4799,7 +4806,7 @@ exports.BattleFormatsData = {
 		tier: "BL3",
 	},
 	tirtouga: {
-		randomBattleMoves: ["shellsmash", "aquajet", "liquidation", "stoneedge", "earthquake", "stealthrock"],
+		randomBattleMoves: ["shellsmash", "aquajet", "liquidation", "stoneedge", "zenheadbutt"],
```

Comment:
keep earthquake and stealth rock

Reply:
see: ice shard shellder

Reply:
one of these (read: zh) would prob have to go